lib/controllers/project.js

```diff
@@ -761,8 +760,31 @@ Project.prototype.detectProjectType = function (path, callback) {
   */
 //-----------------------------------------------------------------------------
 Project.prototype.isSupported = function (srcVersion, runtime) {
+  var imageVersion, majorPass, minorPass, patchPass;
```

Comment:
let's not rewrite `semver`, you can use that tested code by just handling the weird edge case for Meteor:

```
var input = 'METEOR@1.3.2.4'

var srcVersion = input.replace('METEOR@', '')
  .split('.')
  .slice(0, 3)
  .join('.')

// srcVersion === '1.3.2', ready for `semver.gte()`
```